src/app/frontend/replicasetdetail/replicasetdetail.html

```diff
@@ -304,7 +304,7 @@ <h1 flex class="md-title kd-replicasetdetail-app-name">
               <kd-sorted-header currently-selected-column="ctrl.sortEventsBy"
                                 currently-selected-order="ctrl.eventsOrder"
                                 column-name="firstSeen"
-                                tooltip="When the event was seen for the first time">
+                                tooltip="Time that passed since the event was seen for the first time">
```

Comment:
I'm not sure if this sentence is correct. In the table there are dates, not information about time that "passed since". For example `First seen at 1/27/16 2:53 PM` not `12 days`.